### DISCUSSION TITLE
Raspberry Pi Zero availability

### DISCUSSION THREAD

**Discussion by mikenturner** [cardonabits/haxo-hw#18] [General]
Hi All
I'm working out where to source my parts (UK), and I can't seem to find the RPi Zero anywhere.  Does anyone know where I can get one or know when stores are likely to have them in?
Thanks
Mike

**Comment by jcard0na** [comment]
Hi @mikenturner 

There should be new inventory [really soon](https://www.raspberrypi.com/news/supply-chain-update-its-good-news/).  Until then, you can check https://rpilocator.com/?cat=PIZERO to find out who still has stock.  Apparently the Pi Hut, in the UK, has some: https://thepihut.com/products/raspberry-pi-zero-w?variant=31901049749566

**Reply by mikenturner** [reply]
Hi - yes I saw the rpilocator and Pi Hut.  Pi Hut only has the wifi model and I'd prefer the basic (cheaper) one.  As I haven't ordered the PCBs yet I'll wait before ordering the Pi.
Thanks

**Reply by mikenturner** [reply]
As I have now built everything else and just waiting for the Pi Zero availability, I thought I might go for one of the other models.  But first I wanted to check whether the Pi Zero W and/or Pi Zero 2 W are compatible with the haxophone?
Thanks

**Reply by jcard0na** [reply]
Hi Mike,

The Pi Zero W is now widely available and certainly compatible with the haxophone.  I and several others have been using it with good results.

The Pi Zero 2 W should also be compatible based on my review of the [technical specifications](https://datasheets.raspberrypi.com/rpizero2/raspberry-pi-zero-2-w-product-brief.pdf), but no one seems to have tried it yet.  If you go with that, please open a [discussion](https://github.com/jcard0na/haxo-hw/discussions) so we can help you out.  

Cheers,
